repos/exporter/lib/database.ts

```diff
@@ -254,3 +258,14 @@ export const saveIdsOfServicesRequiringAttentionInTheDb = async (
         );
     }
 };
+
+export const removeAllServicesRequiringAttentionIds = async (): Promise<void> => {
+    const serviceQuery = `UPDATE products
+                          SET servicesRequiringAttention = null`;
+
+    try {
+        await executeQuery(serviceQuery, []);
+    } catch (error) {
```

Comment:
FYI you don't need to catch errors in a lambda, you're better off letting them be logged at the top level otherwise you'll need to explicitly log it here so we know what the actual error was if this ever happens